common/test/acceptance/pages/studio/course_info.py

```diff
@@ -1,16 +1,169 @@
 """
 Course Updates page.
 """
-
 from common.test.acceptance.pages.studio.course_page import CoursePage
+from common.test.acceptance.pages.studio.utils import type_in_codemirror, set_input_value
 
 
 class CourseUpdatesPage(CoursePage):
     """
     Course Updates page.
     """
-
     url_path = "course_info"
 
     def is_browser_on_page(self):
-        return self.q(css='body.view-updates').present
+        """
+        Returns whether or not the browser on the page and has loaded the required content
+        """
+        return self.q(css='.handouts-content').present
+
+    def are_course_updates_on_page(self):
+        """
+        Is the course updates content Div loaded on the page
+        """
+        return self.q(css='article#course-update-view.course-updates').present
+
+    def is_new_update_button_present(self):
+        """
+        Checks for the presence of the new update post button
+        """
+        return self.q(css='.new-update-button').present
+
+    def click_new_update_button(self):
+        """
+        Wait for the handouts-content, otherwise the render of the page will clear the new update
+        form when loading content
+        """
+        self.wait_for_element_presence('.handouts-content', '.handouts-content')
```

Comment:
This shouldn't be necessary anymore with is_browser_on_page checking for this element to be present.